test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/Blocks_Utf8_Runtime.codegen.cs

```diff
@@ -0,0 +1,154 @@
+#pragma checksum "TestFiles/IntegrationTests/CodeGenerationIntegrationTest/Blocks_Utf8.cshtml" "{ff1816ec-aa5e-4d10-87f7-6f4963833460}" "1c75b6ede56841d893b94eca2c485c983b807fca"
+// <auto-generated/>
+#pragma warning disable 1591
+[assembly: global::Microsoft.AspNetCore.Razor.Hosting.RazorCompiledItemAttribute(typeof(Microsoft.AspNetCore.Razor.Language.IntegrationTests.TestFiles.TestFiles_IntegrationTests_CodeGenerationIntegrationTest_Blocks_Utf8_Runtime), @"default", @"/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/Blocks_Utf8.cshtml")]
+namespace Microsoft.AspNetCore.Razor.Language.IntegrationTests.TestFiles
+{
+    #line hidden
+    [global::Microsoft.AspNetCore.Razor.Hosting.RazorSourceChecksumAttribute(@"SHA1", @"1c75b6ede56841d893b94eca2c485c983b807fca", @"/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/Blocks_Utf8.cshtml")]
+    public class TestFiles_IntegrationTests_CodeGenerationIntegrationTest_Blocks_Utf8_Runtime
+    {
+        #pragma warning disable 1998
+        public async System.Threading.Tasks.Task ExecuteAsync()
+        {
+#line 2 "TestFiles/IntegrationTests/CodeGenerationIntegrationTest/Blocks_Utf8.cshtml"
+  
+    int i = 1;
+
+#line default
+#line hidden
+            System.ReadOnlySpan<byte> __bytes1 = new byte[]{13, 10};
+            WriteLiteral(__bytes1);
+#line 6 "TestFiles/IntegrationTests/CodeGenerationIntegrationTest/Blocks_Utf8.cshtml"
+ while(i <= 10) {
+
+#line default
+#line hidden
+            System.ReadOnlySpan<byte> __bytes2 = new byte[]{32, 32, 32, 32, 60, 112, 62, 72, 101, 108, 108, 111, 32, 102, 114, 111, 109, 32, 67, 35, 44, 32, 35};
```

Comment:
These spans will be created with zero-copy from the data segment of the dll. The compiler is smart.